spec/queries/get_change_offer_options_spec.rb

```diff
@@ -82,6 +82,22 @@ def set_org_level_make_decisions(training_provider, make_d1, ratifying_provider,
   end
 
   describe '#offerable_courses' do
+    it 'returns only courses which are open on apply and exposed in find' do
```

Comment:
we have removed this logic. now and will return all courses, could we remove this test please?